accepted/2020/or-greater-defines/or-greater-defines.md

```diff
@@ -0,0 +1,218 @@
+# OR_GREATER preprocessor symbols for TFMs
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+Today, projects automatically define `#if` symbols based on the friendly TFM
+representation. This includes both versionless- as well as version-specific
+symbols. For .NET 6, we plan to add `XXX_OR_GREATER` variants that are
+accumulative in nature:
+
+TFM                 | Existing | New
+--------------------|----------|--------------------------------------------
+`net5.0`            | `NET5_0` | `NET5_0_OR_GREATER`
+`net6.0`            | `NET6_0` | `NET6_0_OR_GREATER`, `NET5_0_OR_GREATER`
+
+The goal is to align the semantics between package folder names and `#if`
+without breaking existing code.
+
+The new symbols will make it easier to write code that doesn't need to be
+updated each time the project's `TargetFramework`/`TargetFrameworks` property is
+changed. For example, let's say you want a new code path that is triggered for
+.NET 6.0 and higher. This could either be a new API you want to call or relying
+on a new behavior. Today, you need to write code like that but you need to
+consider what version TFMs your project is targeting. Let's say you target .NET
+Framework 4.6.1, .NET Core 3.1, and .NET 5.
+
+Today, you'd need to write code like this:
+
+```C#
+void M()
+{
+    #if NET461
+        LegacyNetFxBehavior();
+    #elif NETCOREAPP3_1
+        LegacyNetCoreBehavior();
+    #elif NET
+        Net5OrHigherBehavior();
+    #else
+        #error Unhandled TFM
+    #endif
+}
+```
+
+Please note that order is important here. You first handle all the legacy TFMs
+and then you handle the new behavior. To avoid breaking that code once you add
+target .NET 6, you use the versionless TFM for .NET 5 and higher platforms
+(`NET`). However, the code still needs to be revised when you add lower versions
+of existing TFMs.
+
+With this feature, you'll be able to express it like this, which is a lot more
+intuitive:
+
+```C#
+void M()
+{
+    #if NET5_0_OR_GREATER
```

Comment:
I find your lack of taste disturbing, but I succumb to the larger principle of consistency :-)